### DISCUSSION TITLE
Share OSM downloaded maps

### DISCUSSION THREAD

**Discussion by bigkrp** [organicmaps/organicmaps#2791] [Q&A]
If i have already downloaded OSM maps, for example in OsmAnd+ app, can I use them for my Organic Maps app?
Also, may be, there are some opportunity to migrate "saved places" from OsmAnd+ to Organic Maps?

**Comment by biodranik** [comment]
1. No.
2. You can convert GPX to KML, there are some online and offline tools to convert. And then import KML/KMZ into OM. GPX support is planned.
